Makefile

```diff
@@ -1,6 +1,6 @@
 # PREAMBLE
 MIN_PACKER_VERSION=1.6 # for building images
-MIN_TERRAFORM_VERSION=1.0 # for deploying modules
+MIN_TERRAFORM_VERSION=1.2 # for deploying modules
```

Comment:
This is ok, but you should probably update https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/main/pkg/modulewriter/tfversions.go#L19 as well.